invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/stats.html

```diff
@@ -25,138 +25,85 @@
-                {{ _('Downloads in one hour user-sessions are counted only once') }}
-              </p>
-            </td>
-            <td data-label="{{_('All versions')}}" class="right aligned">{{ stats_popup(all_versions.unique_downloads) }}</td>
-            <td data-label="{{_('This version')}}" class="right aligned">{{ stats_popup(this_version.unique_downloads) }}</td>
-          </tr>
-          <tr>
-            <td>
-              {{ _("Downloads") }}
-              <i
-                tabindex="0"
-                role="button"
-                style="position:relative"
-                class="popup-trigger question circle small icon"
-                aria-expanded="false"
-                aria-label="{{ _('More info') }}"
-                data-variation="mini inverted"
-              >
-              </i>
-              <p role="tooltip" class="popup-content ui flowing popup transition hidden">
-                {{ _('Total downloads') }}
-              </p>
-            </td>
-            <td data-label="{{_('All versions')}}" class="right aligned">{{ stats_popup(all_versions.downloads) }}</td>
-            <td data-label="{{_('This version')}}" class="right aligned">{{ stats_popup(this_version.downloads) }}</td>
-          </tr>
-          <tr>
-            <td>
-              {{ _("Data volume") }}
-              <i
-                tabindex="0"
-                role="button"
-                style="position:relative"
-                class="popup-trigger question circle small icon"
-                aria-expanded="false"
-                aria-label="{{ _('More info') }}"
-                data-variation="mini inverted"
-              >
-              </i>
-              <p role="tooltip" class="popup-content ui flowing popup transition hidden">
-                {{ _('Total data volume') }}
-              </p>
-            </td>
-            {%- set binary_sizes = not config.APP_RDM_DISPLAY_DECIMAL_FILE_SIZES %}
+        <td data-label="{{_('All versions')}}" class="right aligned">{{ all_versions.data_volume|filesizeformat(binary=binary_sizes) }}</td>
+        <td data-label="{{_('This version')}}" class="right aligned">{{ this_version.data_volume|filesizeformat(binary=binary_sizes) }}</td>
+      </tr>
+    </tbody>
+  </table>
 
-            <td data-label="{{_('All versions')}}" class="right aligned">{{ all_versions.data_volume|filesizeformat(binary=binary_sizes) }}</td>
-            <td data-label="{{_('This version')}}" class="right aligned">{{ this_version.data_volume|filesizeformat(binary=binary_sizes) }}</td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-  </div>
 
-  <p class="m-auto rel-mt-1 rel-mb-1 text-muted">
-    <i>{{ _("All statistics are updated daily.")}} <a href="/help/statistics">{{ _("Read more") }}...</a></i>
+  <p class="text-muted">
+    <a href="/help/statistics">{{ _("More info on how stats are collected.") }}...</a>
```

Comment:
Without the dot?
```suggestion
    <a href="/help/statistics">{{ _("More info on how stats are collected") }}...</a>
```
Will the static page `/help/statistics` always be there?

Reply:
yes, it is hardcoded for now